src/video-player.utils.js

```diff
@@ -121,7 +120,7 @@ export const overrideDefaultVideojsComponents = () => {
     children.splice(children.indexOf('progressControl'), 0, 'spacer', 'progressControlEventsBlocker');
 
     // Add 'play-previous' and 'play-next' buttons around the 'play-toggle'
```

Comment:
This comment is not longer relevant - thats why i general i do not like comments, they very easily get outdated 
```suggestion
```